app/views/shared/_navbar.html.erb

```diff
@@ -4,9 +4,13 @@
     <span>.</span>
   </div>
 
-  <%= form_with url: "#", class: "max-content" do |form| %>
-    <%= form.text_field :search, placeholder: "What are you looking for ?" %>
-    <%= button_tag class:"btn-gold max-content" do %>
+  <%= form_with url: arts_path, method: :get, class: "d-flex max-content" do %>
+    <%= text_field_tag :query,
+      params[:query],
+      class: "form-control",
+      placeholder: "Find an art piece"
+    %>
+    <%= button_tag( class: "btn-gold max-content") do %>
```

Comment:
d-flex and form-control are bootstrap classes, aren't they ?

Is there a reason you added parentheses to the button_tag ?